scripts/services/sponsors.js

```diff
@@ -0,0 +1,48 @@
+angular
+.module('khe')
+.factory('Sponsors', ['$http', 'socketFactory', 'User', function ($http, socket, User) {
+
+    var Sponsors = function() {
+
+        var self = this;
+        var user = new User();
+        
+        self.list = function() {
+            var req = {
+                method: 'GET',
+                url: config.api + '/sponsors'
+            };
+            return $http(req);
+        };
+
+        self.addSponsor = function(newSpons) {
+            var req = user.authorize({
+                method: 'POST',
+                url: config.api + '/sponsors',
+                data: newSpons
+            });
+            return $http(req);
+        };
+
+        self.update = function(spons) {
+            var req = user.authorize({
+                method: 'POST',
+                url: config.api + '/sponsors/' + spons._id + '/update',
```

Comment:
Backend has been changed to follow CRUD, remove '/update' from the end of the url